libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/location/ConstantVelocityInterpolatorTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.mapbox.navigation.ui.maps.internal.location
+
+import com.mapbox.geojson.Point
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.math.pow
+
+@RunWith(RobolectricTestRunner::class)
+class ConstantVelocityInterpolatorTest {
+
+    @Test
```

Comment:
There are no tests for:
1) An empty array;
2) Array with single point;
3) Array with 2 identical points (so that the distance is zero).

Reply:
> There is no need for such tests.

why? shouldn't we handle such edge cases and have tests for them?

Reply:
I also don't see why we shouldn't test it.

Reply:
Done. Added more tests that verify above cases.